lib/chef/resource/file/verification.rb

```diff
@@ -122,6 +122,16 @@ def verify_registered_verification(path, opts)
           v = verification_class.new(@parent_resource, @command, @command_opts, &@block)
           v.verify(path, opts)
         end
+
+        def to_s
+          if @block
+            '<Proc>'
```

Comment:
should probably at least print out Proc#source_location?   seems to be a bit annoying to get text out of a Proc...